enterprise/coderd/provisionerdaemons.go

```diff
@@ -227,7 +227,7 @@ func (api *API) provisionerDaemonServe(rw http.ResponseWriter, r *http.Request)
 		Provisioners:          daemon.Provisioners,
 		Telemetry:             api.Telemetry,
 		Auditor:               &api.AGPL.Auditor,
-		TemplateScheduleStore: &api.AGPL.TemplateScheduleStore,
+		TemplateScheduleStore: api.AGPL.TemplateScheduleStore,
```

Comment:
nit: is it intentional change?

Reply:
Yes, `provisionerdserver` was using a `*atomic.Pointer[T]` so we had to use `&` before since `coderd` was only using `atomic.Pointer[T]`. `coderd` now uses `*atomic.Pointer[T]` so we just copy the pointer here.